preview.netcdf/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8
+
+WORKDIR /extractor
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+
+COPY netcdfutils.py ncsa.geo.netcdf.extractor.py extractor_info.json ./
+CMD python ncsa.geo.netcdf.extractor.py
```

Comment:
No need to pass in CLOWDER_VERSION as parameters here?

Reply:
this is added